model-utils/src/main/java/io/github/ygojson/model/utils/data/utils/LocalizationUtils.java

```diff
@@ -0,0 +1,287 @@
+package io.github.ygojson.model.utils.data.utils;
+
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+import io.github.ygojson.model.data.Card;
+import io.github.ygojson.model.data.Set;
+import io.github.ygojson.model.data.definition.CardText;
+import io.github.ygojson.model.data.definition.SetInfo;
+import io.github.ygojson.model.data.definition.localization.CardLocalizedData;
+import io.github.ygojson.model.data.definition.localization.Language;
+import io.github.ygojson.model.data.definition.localization.SetLocalizedData;
+
+public class LocalizationUtils {
+
+	/**
+	 * Gets the localized {@link CardText} for the card.
+	 * <br>
+	 * Note that for {@link Language#EN} a populated {@link CardText} is returned
+	 * where changes will not affect the set.
```

Comment:
Add a warning (maybe at the class level) that this assumes that the main-language haven't changed.